aggregators/aggregate.js

```diff
@@ -19,19 +19,38 @@
 
 class Aggregate {
 
+  /**
+   * @throws {Error}
+   * @return {string} The name for this aggregation.
+   */
   static get name() {
     throw new Error('Aggregate name must be overridden');
   }
 
+  /**
+   * @throws {Error}
+   * @return {!Criteria} The criteria for this aggregation.
+   */
   static get criteria() {
     throw new Error('Aggregate criteria must be overridden');
   }
 
+  /**
+   * @private
+   * @param  {!Array<!AuditResult>} results
+   * @param  {!Criteria} expected
+   * @return {!Array<!AuditResult>}
+   */
   static _filterResultsByAuditNames(results, expected) {
     const expectedNames = Object.keys(expected);
     return results.filter(r => expectedNames.indexOf(r.name) !== -1);
   }
 
+  /**
+   * @private
+   * @param  {!Criteria} expected
```

Comment:
ugh, I really dislike the aligned braces in JSDocs thing. We should pick a style here :)